core/templates/reports/language_batch_pages.html

```diff
@@ -50,7 +50,7 @@
     <td>{{ p.reel__number }}</td>
     <td><a href="{% url 'chronam_language_title_pages' language batch p.issue__title__lccn %}">{{ p.issue__title__lccn }}</a></td>
     <td>{{ p.issue__date_issued }}</td>
-    <td><a href="{% url 'chronam_page p.issue__title__lccn' p.issue__date_issued p.issue__edition p.sequence  %}">{{ p.sequence }}</a></td>
+    <td><a href="{% url 'chronam_page' p.issue__title__lccn p.issue__date_issued p.issue__edition p.sequence  %}">{{ p.sequence }}</a></td>
```

Comment:
I think the defect described in issue #120 is not fixed by this change. We should fix the URL here: https://github.com/LibraryOfCongress/chronam/blob/chronam_solr_4_10/core/templates/reports/language_batches.html#L48